Simplified instructions and added some do not(s) (#4)
one-drop/onedrop-android-code-challenge: README.md

# One Drop Android Coding Challenge

> At One Drop we take great pride in the quality of our product and our engineering team’s software craft. As such, we request that engineer candidates submit a code challenge for us to make a general assessment of their expertise with the Android platform and software development as a whole.

Reviewer: ```suggestion
At One Drop we take great pride in the quality of our product and our engineering team’s software craft. As such, we request that engineer candidates submit a code challenge to help us assess their expertise with the Android platform and software development as a whole.
```

### Requirements

> 1. Using the [One Call API](https://openweathermap.org/api/one-call-api), retrieve the weather information for New York. In the spirit of time, use a static latitude/longitude coordinate for the city. We have created for you a `WeatherApi` Retrofit interface with the required endpoint. From this API response, we expect the following set of data to be shown on the main/landing screen:

Reviewer: ```suggestion
1. Retrieve the weather information for New York City using the [One Call API](https://openweathermap.org/api/one-call-api). To save time, you can hardcode the city's latitude/longitude coordinate. In the project, you'll find an existing `WeatherApi` implementation with the required endpoint. From this API response, we expect the following set of data to be shown on the main/landing screen:
```

4. Project must be architected using [Model-View-Presenter](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93presenter) (MVP)
5. Feel free to use any tools or libraries you are comfortable with.

> 6. Include a README.md to your project that includes the following information:

Reviewer: ```suggestion
6. Include a README.md to your project that includes the following:
```

    * A brief explanation of your code architecture or libraries employed
    * Sample screenshots of your app.

>     * Optional: A “future work” section where you can tell the team about feature or code optimizations that you would have done in the app if you had more time allocated for this challenge.

Reviewer: ```suggestion
    * Optional: A “future work” section where you describe additional features or code optimizations that you would have added to the app if you had more time allocated for this challenge.
```

### Bonus Points
* Include a few unit tests
* Use of RxJava for all your network requests

> * Use Android's location API to use the device's current location (i.e. latitude/longitude coordinates) instead of a static location.

Reviewer: ```suggestion
* Use Android's location API to display weather data for the device's current location rather than hardcoding the latitude/longitude coordinates.
```

### Submitting your work
Once you've completed, push your project to your preferred Git host (Github, GitLab, Bitbucket, etc) and send us the repo link so we can take a look.

* If sharing a public git project is not possible, sending an archive (i.e. zip) of the project is also acceptable.

> * Do not fork this repo
> * Do not submit a pull-request

Reviewer: ```suggestion
* Please do *NOT* fork this repo or create a pull request for your submission.
```
